tools/lasdump/CMakeLists.txt

```diff
@@ -4,13 +4,15 @@ cmake_minimum_required(VERSION 3.13)
 add_executable(lasdump
     Dumper.cpp
     Header.cpp
+    ${PDAL_UTIL_DIR}/VSIIO.cpp
     ${PDAL_UTIL_DIR}/Bounds.cpp
     ${PDAL_UTIL_DIR}/FileUtils.cpp
     ${PDAL_UTIL_DIR}/Utils.cpp
 )
 
 target_link_libraries(lasdump PRIVATE
     ${PDAL_LAZPERF_LIB_NAME}
+    ${GDAL_LIBRARY}
```

Comment:
`lasdump` was never intended to link any thing external, but this isn't a very important utility.